docs/API.md

```diff
@@ -0,0 +1,26 @@
+# API
+
+## Socket events
+
+### Client
+
+Client-side API sends socket events to the server.
+
+sendToServer.js handles events dispatching.
+
+The serverObject provides methods to use :
+
+- `serverObject.sendNewUser()` returns a client ID (String)
```

Comment:
That's fine for now, but nowhere else in the application is defined what a "client" is. Prefer "snake" or "player"